frontend/tests/features/wheresMyGene/tissueAutoExpand.test.ts

```diff
@@ -106,7 +131,182 @@ describe("WMG tissue auto-expand", () => {
     await checkTissues(page, ["lung"], []);
     await checkCellTypes(page, cells);
   });
+
+  /**
+   * Tissue auto expansion - cross filter with Sex filter selected
+   * Filter Female from Sex filter. Filter Abdomen and Blood from the Tissue
+   * filter - those should be expanded, Add Cell type filtering,
+   * only B Cell type should appear and expanded.
+   */
+  test("Tissue auto expansion - cross filter with Sex filter selected", async ({
+    page,
+  }) => {
+    const tissues = FILTERED_TISSUES.slice(0, 1);
+    await loadPageAndTissues(page);
+    await filterSelection(
+      page,
+      SEX_FILTER_TEST_ID,
+      SEX_FILTER_LABEL,
+      SEX_FILTER_SELECTION
+    );
+    await filterTissues(page, tissues);
+    await filterCellType(page, 1);
+    await checkTissues(page, tissues);
+  });
+
+  /**
+   * Tissue auto expansion - cross filter with Sex filter, check expansion
+   * Filter 3 Tissues ["abdomen", "axilla", "blood"]. Collapse Abdomen. Select
+   * Female from the Sex filter. Tissue filter should now only have Abdomen and
+   * Blood selected. Only Abdomen and Blood should be visible. Abdomen should
+   * remain collapsed. Remove Sex filter. Tissue filter should now only have
+   * Abdomen and Blood selected. Only Abdomen and Blood should be visible.
+   * Abdomen should remain collapsed.
+   */
+  test("Tissue auto expansion - cross filter with Sex filter, check expansion", async ({
+    page,
+  }) => {
+    const filteredTissues = FILTERED_TISSUES.slice();
+    const expandedTissues = filteredTissues.slice();
+    await loadPageAndTissues(page);
+    await filterTissues(page, filteredTissues);
+    await collapseTissue(page, filteredTissues[0]);
+    expandedTissues.splice(0, 2);
+    await filterSelection(
+      page,
+      SEX_FILTER_TEST_ID,
+      SEX_FILTER_LABEL,
+      SEX_FILTER_SELECTION
+    );
+    filteredTissues.splice(1, 1); // remove axilla
```

Comment:
instead of mutating the array, should we just hard code expected states, so we don't have to track how the array is being mutated in our heads?

Reply:
Something like this!

```ts
/**
   * Tissue auto expansion - cross filter with Sex filter, check expansion
   * Filter 3 Tissues ["abdomen", "axilla", "blood"]. Collapse Abdomen. Select
   * Female from the Sex filter. Tissue filter should now only have Abdomen and
   * Blood selected. Only Abdomen and Blood should be visible. Abdomen should
   * remain collapsed. Remove Sex filter. Tissue filter should now only have
   * Abdomen and Blood selected. Only Abdomen and Blood should be visible.
   * Abdomen should remain collapsed.
   */
  test("Tissue auto expansion - cross filter with Sex filter, check expansion", async ({
    page,
  }) => {
    await loadPageAndTissues(page);
    await filterTissues(page, FILTERED_TISSUES);
    await collapseTissue(page, FILTERED_TISSUES[0]);

    await filterSelection(
      page,
      SEX_FILTER_TEST_ID,
      SEX_FILTER_LABEL,
      SEX_FILTER_SELECTION
    );

    /**
     * Once female filter is applied, we expect only the following tissues to be present
     */
    const EXPECTED_FILTERED_TISSUES_WITH_SEX_FILTER = ["abdomen", "blood"];

    await expect(page.getByTestId(TISSUE_NODE_TEST_ID)).toHaveCount(2);

    const EXPECTED_EXPANDED_TISSUES = ["blood"];

    await checkTissues(
      page,
      EXPECTED_FILTERED_TISSUES_WITH_SEX_FILTER,
      EXPECTED_EXPANDED_TISSUES
    );

    /**
     * Also, I think the following steps aren't tested yet!
     * Remove Sex filter. Tissue filter should now only have Abdomen and Blood selected. Only Abdomen and Blood should be visible.
     * Abdomen should remain collapsed
     */
  });
```